hubblestack/extmods/grains/splunkindex.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+
+import salt.modules.config
+
+__salt__ = {'config.get': salt.modules.config.get}
+
+
+def splunkindex():
+    '''
+    Return splunk index from config file in grain
+    '''
+    # Provides:
+    #   splunkindex
+    grains = {}
+    index = __salt__['config.get']('hubblestack:returner:splunk:index', default='unknown')
```

Comment:
Let's either default this to `None`, or just not define it at all if we can't find that value.